### PR TITLE
Create 2024-03-11-transportation-round-up.md

### PR DESCRIPTION
**Pull request by laneymangan**
@AnthonyRollins can you take a look when you have a minute? this is a work in progress (I still want to add photos, tweak a few things, etc) but figured I'd go ahead and pull you in now so you have access for whenever you find some time

### CODE REVIEW

**Comment by angela-tran** [comment]
@laneymangan I'm not sure if you were wanting your post to be hidden, but I noticed your post isn't showing up on the preview site because its date is set in the future.

From the Netlify deploy logs:
```
4:11:47 PM: $ jekyll build
4:11:47 PM: Configuration file: /opt/build/repo/_config.yml
4:11:47 PM:             Source: /opt/build/repo
4:11:47 PM:        Destination: /opt/build/repo/_site
4:11:47 PM:  Incremental build: disabled. Enable with --incremental
4:11:47 PM:       Generating...
4:11:47 PM:           Skipping: _posts/2024-03-11-transportation-round-up.md has a future date
4:11:47 PM:                     done in 0.308 seconds.
4:11:47 PM:  Auto-regeneration: disabled. Use --watch to enable.
4:11:47 PM: ​
4:11:47 PM: (build.command completed in 802ms)
4:11:47 PM: ​
4:11:48 PM: (Netlify Build completed in 1.1s)
4:11:48 PM: Section completed: building
4:11:53 PM: Finished processing build request in 51.912s
```

The part to note is this line: `Skipping: _posts/2024-03-11-transportation-round-up.md has a future date`.

**Comment by laneymangan** [comment]
@angela-tran I changed the date to today, did that fix it?

**Comment by angela-tran** [comment]
> @angela-tran I changed the date to today, did that fix it?

Yes, very nice!

File: _posts/2024-03-11-transportation-round-up.md
Comment (AnthonyRollins) [inline _posts/2024-03-11-transportation-round-up.md:17]: Replace `has to say` to `says`

File: _posts/2024-03-11-transportation-round-up.md
Comment (AnthonyRollins) [inline _posts/2024-03-11-transportation-round-up.md:49]: Let's remove the brackets and add italics to the sentence. It would be good for this sentence to be its own paragraph. It's not super critical; I think the space highlights your internal thoughts.

File: _posts/2024-03-11-transportation-round-up.md
Comment (AnthonyRollins) [inline _posts/2024-03-11-transportation-round-up.md:7]: Add to excerpt: `The Compiler team's favorite transit stories from the past year`

**Review by AnthonyRollins — CHANGES_REQUESTED** [review]
I have a few changes, but everything looks good. Let me know if you have any questions.

**Comment by laneymangan** [comment]
@compilerla/engineering (cc @angela-tran) ready for approval and merging! Let me know if there are any code errors or anything else needed from me. Thank you!

**Comment by angela-tran** [comment]
@laneymangan I think the `/assets/blog/jessica-sullivan.jpg` file isn't being used anywhere, so can you delete that one? You should be able to do that from the "Files changed" tab in this PR and use "Delete file".

image

**Comment by laneymangan** [comment]
@angela-tran hm, doesn't look like I have permissions to delete 
image

**Comment by angela-tran** [comment]
How about from here: https://github.com/laneymangan/compiler.la/blob/patch-2/assets/blog/jessica-sullivan.jpg?

image

**Comment by laneymangan** [comment]
@angela-tran that worked!

**Comment by angela-tran** [comment]
@laneymangan I'm glad that one lets you delete! I think you need to commit the deletion though 😅

**Comment by laneymangan** [comment]
@angela-tran whoops. done!

**Review by angela-tran — APPROVED** [review]
Awesome work @laneymangan !! This looks good to me

**Comment by angela-tran** [comment]
@AnthonyRollins Since you had requested changes on this PR, merging is blocked until you approve it as well

image

**Review by AnthonyRollins — APPROVED** [review]
Looks good

**Pull request merged**
